automation/taskcluster/lib/tasks.py

```diff
@@ -597,6 +601,7 @@ def _craft_raptor_task(
             dependencies=[signing_task_id],
             name=task_name,
             description=description,
+            routes=['notify.email.perftest-alerts@mozilla.com.on-failed'],
```

Comment:
Can we also send this to fenix-eng-notifications@mozilla.com?

Reply:
This will email out on intermittents too. Are we sure we want to put them in the loop?

Reply:
Hmm.. good point on the noise this will generate. My concern is that we're not actively sheriffing these jobs. I want to be informed, but my team isn't going to be investigating these failures.

Reply:
We can start with perftest-alerts, and revisit.